src/components/Todo/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import EditionForm from './EditionForm';
+import DueDate from './DueDate';
+import Dropdown, { DropdownItem } from '../library/Dropdown';
+
+export default class Todo extends React.Component {
+
+  render() {
+    const { todo, editing, onUpdate, onCancelEdit, onEdit, onDelete } = this.props;
+
+    if (editing) {
+      return (
+        <span className="todo-container">
+          <EditionForm
+            todo={todo}
+            onUpdate={onUpdate}
+            onCancel={onCancelEdit}
+          />
+        </span>
+      )
+    }
+
+    return (
+      <span className="todo-container">
+
+        <span
+          style={{
+            textDecoration: todo.active ? "none" : "line-through"
+          }}
+          onClick={() => onUpdate(todo.id, { active: !todo.active})}
+        >
+          {todo.name}
+        </span>
+
+
+
+        <DueDate dueDate={todo.dueDate}/>
+
+        <Dropdown>
+          <DropdownItem onClick={() => onEdit(todo)}>Edit</DropdownItem>
+          <DropdownItem
+            onClick={() => onDelete(todo.id)}
+            style={{ color: "red" }}
```

Comment:
It works indeed ! But it would have been better to use Bulma class so that the colors that you use throughout the app remain consistent.
The good answer would be :
```jsx
<DropdownItem
  onClick={() => onDelete(todo.id)}
>
  <span className="has-text-danger">Delete</span>
</DropdownItem>
```

Notice that you can totally pass JSX to a component property. In this case the implicit property is "children", but you can pass JSX to any other property. 
Another example of this:
```jsx
<Dropdown
  trigger={<Button color="success">Click me to open the dropdown</Button>}
>
  <DropdownItem>One link</DropdownItem>
  <DropdownItem>Another link</DropdownItem>
</Dropdown>
```
In this case the component Dropdown receives JSX through its property `children` and also through `trigger`. The property trigger allow the developper to put whatever he wants as a trigger for a dropdown.

Reply:
Take a look at this example in the doc
https://reactjs.org/docs/composition-vs-inheritance.html#containment